proto/import_sstpb.proto

```diff
@@ -141,6 +145,8 @@ message DownloadRequest {
     RewriteRule rewrite_rule = 13 [(gogoproto.nullable) = false];
 
     backup.StorageBackend storage_backend = 14;
+
+    encryptionpb.EncryptionConfig encryption_config = 15;
```

Comment:
Is this needed for Upload and Ingest requests?

Reply:
AFAIK BR only use the download API. Is it true?

Reply:
BR uses both Download and Ingest.

Upload is used by TiKV Importer at the moment.

Reply:
Just to confirm, `download` is for pulling files from storage,  `ingest` is to ask tikv to ingest the downloaded file, is that correct? In that case only `download` needs the config to decrypt the files. And this PR is for BR only.

Reply:
i see.